report.Rmd

```diff
@@ -141,7 +151,7 @@ fit_tobit <- function(data, predictors, to_predict = "Got", interaction = FALSE,
     if (interaction) {
         model_formula <- sprintf("%s+%s", model_formula, paste(paste("MD *", predictors[3:length(predictors)]), collapse = " + "))
     }
-    model <- tobit(as.formula(model_formula), data = data, left = limits$left, right = limits$right)
+    model <- tobit(as.formula(model_formula), data = df, left = limits$left, right = limits$right)
```

Comment:
It will not work - the functions takes argument `data`, not `df`. I will suggest leaving previous version.